src/wp-admin/includes/file.php

```diff
@@ -1563,6 +1563,34 @@ function wp_trusted_keys() {
 	return apply_filters( 'wp_trusted_keys', $trusted_keys );
 }
 
+/**
+ * Test whether the given file is a valid Zip file.
```

Comment:
```suggestion
 * Determines whether the given file is a valid ZIP file.
```

- Third-person singular verb.
- The format is `ZIP`.

That's strike 1.